sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/LongExemplar.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.data;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.api.common.Attributes;
+import javax.annotation.concurrent.Immutable;
+
+/** An {@link Exemplar} with {@code long} measurments. */
+@Immutable
+@AutoValue
+public abstract class LongExemplar implements Exemplar {
+
+  /**
+   * Construct a new exemplar.
+   *
+   * @param filteredAttributes The set of {@link Attributes} not already associated with the {@link
+   *     PointData}.
+   * @param recordTimeNanos The time when the sample qas recorded in nanoseconds.
+   * @param spanId (optional) The associated {@code SpanId}
```

Comment:
Perhaps we can `compileOnly(":api:all")` to be able to `{@link SpanId}`. Otherwise, having this in `@code` doesn't seem particularly helpful

Reply:
Removed `@code` for now.  We can revisit if we want to document any specific string formatting concerns related to using SpanId/TraceId